harry_is_rational.py

```diff
@@ -0,0 +1,23 @@
+import os
+import re
+import codecs
+
+
+def rename_files(path):
+    list_of_file = os.listdir(path)
+    for entry in list_of_file:
+        full_path = os.path.join(path, entry)
+        codecs.register_error("strict", codecs.ignore_errors)
```

Comment:
While the builtin [open()](https://docs.python.org/3/library/functions.html#open) and the associated [io](https://docs.python.org/3/library/io.html#module-io) module are the recommended approach for working with encoded text files, this module provides additional utility functions and classes that allow the use of a wider range of codecs when working with binary files:
from here - https://docs.python.org/3/library/codecs.html

Reply:
why did you use codecs?